### DISCUSSION TITLE
Question: how does Renovate treat 0.x -> 0.y updates by default?

### DISCUSSION THREAD

**Discussion by FernandesMF** [renovatebot/renovate#30925] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

None

### Please tell us more about your question or problem

Hello, Renovate team!

How does Renovate treat 0.x -> 0.y updates by default? Are they detected as major, or minor?

I'm asking because I believe the convention is to treat them as major (since v0 is changing rapidly and things break often), but in this example Renovate treats them as minor:
https://docs.renovatebot.com/faq/#renovates-default-behavior-for-majorminor-releases

In case they are treated as minor, is there a config that could tweak that?

Thanks!

**Comment by rarkins** [comment, marked as answer]
A major update is one where the major version changes. 0.x -> 0.y is not a major update, but it may be a _breaking_ update. Renovate treats v0 -> v0 updates as minor because that's what they are. You should treat them as potentially breaking though. If you want to do things like exclude v0 updates from a non-major group then you can use the `allowedVersions` config in `packageRules`.

**Reply by FernandesMF** [reply]
Thank you very much, @rarkins!
